compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -1320,8 +1335,9 @@ static LogicalResult setRootConfig(func::FuncOp entryPointFn,
 static LogicalResult setRootConfig(func::FuncOp entryPointFn,
                                    IREE::LinalgExt::FftOp fftOp) {
   assert(!getLoweringConfig(fftOp) && "expected lowering_config is not set");
+  DistributionHeuristicConfig config;
   SmallVector<int64_t> distTileSizes =
-      getDefaultDistributedLevelTileSizes(fftOp);
+      getDefaultDistributedLevelTileSizes(fftOp, config);
```

Comment:
Can be `getDefaultDistributedLevelTileSizes(fftOp, DistributionHeuristicConfig{});`?

Reply:
good point, thanks!